test/api-helper/project/fileCopy.test.ts

```diff
@@ -48,4 +51,89 @@ describe("fileCopy", () => {
         assert(!(await recipient.getFile(filesToSteal[0].donorPath)));
         assert(!!(await recipient.getFile(filesToSteal[0].recipientPath)));
     }).timeout(5000);
+
+    it("should copy a subset of files from donor project according to glob", async () => {
+        const donorProject = InMemoryProject.of(
+            {path: "/", content: "root file"},
```

Comment:
It is not necessary to "add" directories.

Reply:
If you want a file to not be present in the donor and not the recipient, use something like `"/x"`.